_data/sitetext.yml

```diff
@@ -14,6 +14,11 @@ en: &DEFAULT_EN
     title:
     text:
 
+  about:
+    title: "Clinical Scientific Computing"
+    image: assets/img/csc-logo-text.svg
```

Comment:
It looks odd to have the logo here as well in the navbar. It should just be in one place.

Reply:
Disagree – looks good on monai.io, for instance, and think it just reinforces the brand.

Reply:
Fair point.